### PR TITLE
Not Ready for Merge: Allow routes to be disabled

### PR DESCRIPTION
**Pull request by robkilby**
We use a heavily namespaced application consisting of many different engines and needed a way to customise the routes without leaving the default ones hanging around.

This PR adds a simple configuration utility and initializer file. This is my first PR so hopefully everything is OK for you.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
-Rails.application.routes.draw do
```
Comment (michaelglass) [inline config/routes.rb]: please test this functionality and we'll accept the pr, thanks!

Comment (robkilby) [inline config/routes.rb]: Great, I have a few more commits to add to this for your review, and then I'll add tests for you.

**Comment by michaelglass** [comment]
ping, @robkilby

**Comment by robkilby** [comment]
Sorry for going quiet - had a busy few months and completely forgot about this PR. I've pushed a few more updates and am working on tests for the new functionality at the moment.

**Comment by michaelglass** [comment]
thanks1

**Pull request closed without merging**